share/index.html

```diff
@@ -28,7 +28,7 @@
 
```

Comment:
Fixed. I did a semi-hacky quick fix. We'll have to revisit the dashboard when we have more time.